palo_alto_cortex_xdr/plugin.spec.yaml

```diff
@@ -0,0 +1,295 @@
+
+types:
+  isolation_result:
+    action_id:
+      title: "Action ID"
+      type: integer
+      description: "Action ID"
+      required: false
+    status:
+      title: "Status"
+      type: string
+      description: "Status"
+      required: false
+    endpoints_count:
+      title: "Endpoints Count"
+      type: integer
+      description: "Endpoints count"
+      required: false
+  endpoint:
+    alias:
+        title: "Alias"
+        type: string
+        description: "Alias"
+        required: false
+    content_version:
+        title: "Content Version"
+        type: string
+        description: "Content version"
+        required: false
+    domain:
+        title: "Domain"
+        type: string
+        description: "Domain"
+        required: false
+    endpoint_id:
+        title: "Endpoint ID"
+        type: string
+        description: "Endpoint ID"
+        required: false
+    endpoint_name:
+        title: "Endpoint Name"
+        type: string
+        description: "Endpoint name"
+        required: false
+    endpoint_status:
+        title: "Endpoint Status"
+        type: string
+        description: "Endpoint status"
+        required: false
+    endpoint_type:
+        title: "Endpoint Type"
+        type: string
+        description: "Endpoint type"
+        required: false
+    endpoint_version:
+        title: "Endpoint Version"
+        type: string
+        description: "Endpoint version"
+        required: false
+    first_seen:
+        title: "First Seen"
+        type: integer
+        description: "First seen"
+        required: false
+    install_date:
+        title: "Install Date"
+        type: integer
+        description: "Install date"
+        required: false
+    installation_package:
+        title: "Installation Package"
+        type: string
+        description: "Installation package"
+        required: false
+    ip:
+        title: "IP"
+        type: "[]string"
+        description: "IP"
+        required: false
+    is_isolated:
+        title: "Is Isolated"
+        type: string
+        description: "Is isolated"
+        required: false
+    last_seen:
+        title: "Last Seen"
+        type: integer
+        description: "Last seen"
+        required: false
+    operational_status:
+        title: "Operational Status"
+        type: string
+        description: "Operational status"
+        required: false
+    operational_status_description:
+        title: "Operational Status Description"
+        type: string
+        description: "Operational status description"
+        required: false
+    os_type:
+        title: "OS Type"
+        type: string
+        description: "OS type"
+        required: false
+    scan_status:
+        title: "Scan Status"
+        type: string
+        description: "Scan status"
+        required: false
+    users:
+        title: "Users"
+        type: "[]string"
+        description: "Users"
+        required: false
+
+connection:
+  fqdn:
```

Comment:
This should be called `url` to match style of other plugins

Reply:
title should be URL too

Reply:
Disagree, all the cortex documentation and examples say FQDN. If a user is using their docs to get this information, they may put in the dashboard URL instead of the FQDN

Reply:
Should follow our style. Does this work if a user passes in `example.com`?

Reply:
Fixed